src/Antiforgery/src/Internal/AntiforgeryOptionsSetup.cs

```diff
@@ -35,12 +35,8 @@ public void Configure(AntiforgeryOptions options)
 
         private static string ComputeCookieName(string applicationId)
         {
-            using (var sha256 = CryptographyAlgorithms.CreateSHA256())
-            {
-                var hash = sha256.ComputeHash(Encoding.UTF8.GetBytes(applicationId));
-                var subHash = hash.Take(8).ToArray();
-                return WebEncoders.Base64UrlEncode(subHash);
-            }
+            byte[] fullHash = SHA256.HashData(Encoding.UTF8.GetBytes(applicationId));
```

Comment:
Since you're here, may as well improve the allocations here.

```C#
var source = Encoding.UTF8.GetBytes(applicationId);
Span<byte> target = stackalloc byte[32];
SHA256.HashData(source, target);
WebEncoders.Base64UrlEncode(target.Slice(0, 8));
```

Reply:
This code is called exactly once at app start. Wasn't worth complicating the code for so little gain.

Reply:
that's fair